docs/docs/reference/other-new-features/opaques.md

```diff
@@ -28,19 +29,22 @@ object Logarithms {
 }
 ```
 
-This introduces `Logarithm` as a new type, which is implemented as `Double` but is different from it. The fact that `Logarithm` is the same as `Double` is only known in the scope where
-`Logarithm` is defined which in this case is object `Logarithms`.
-
-The public API of `Logarithm` consists of the `apply` and `safe` methods that convert from doubles to `Logarithm` values, an extension method `toDouble` that converts the other way,
-and operations `+` and `*` on logarithm values. The implementations of these functions
-type-check because within object `Logarithms`, the type `Logarithm` is just an alias of `Double`.
+This introduces `Logarithm` as a new abstract type, which is implemented as `Double`. 
+The fact that `Logarithm` is the same as `Double` is only known in the scope where
+`Logarithm` is defined which in the above example corresponds to the object `Logarithms`.
+Or in other words, within the scope it is treated as type alias, but this is opaque to the outside world 
+where in consequence `Logarithm` is treated as own type and has nothing to do with `Double`.
```

Comment:
```suggestion
where in consequence `Logarithm` is treated as its own type and has nothing to do with `Double`.
```
(or perhaps "is seen as an abstract type")

Reply:
Thinking about it, is it really an abstract type? I thought an abstract type is:
- abstract and final
- thus there is no way to instantiate it

Thoughts?

Reply:
Yes, they're really equivalent to abstract types (as in `object Foo { type A }`) from outside.

Reply:
Right, I had something else in mind with abstract type. Makes sense, I am going to update the PR